github-actions/oss-readme-template/src/readme.ts

```diff
@@ -137,6 +145,15 @@ export function generateReadmeStringFromTemplateString(
       stabilityNotes: stabilityNotes,
     };
     ossHeader += nunjucks.renderString(sdkHeaderTemplate, sdkHeaderContext);
+
+    const sdkUsageExamplePathTemplate = OSS_SDK_USAGE_EXAMPLE_PATH_TEMPLATE;
+    const usageExamplePathContext = {
+      usageExamplePath: options.usageExamplePath,
+    };
+    ossUsageExamplePath = nunjucks.renderString(
+      sdkUsageExamplePathTemplate,
+      usageExamplePathContext
+    );
```

Comment:
So I think you would replace this whole new block of code with something more like this:

```
const usageExampleCode = fs.readFileSync(usageExamplePath);
```

So you will actually be slurping in the contents of some file that has code in it.  For the tests, you will need to create a test file in the tests dir (alongside the test templates probably) that is called `usage.ts` or something, and just put some "hello world" code in it.